src/AccessControl.spec.ts

```diff
@@ -14,14 +14,46 @@ describe('Access Control', () => {
     ACWithoutGrants = new AccessControl();
   });
 
-  it('`getRoles()` should return the data passed in the constructor', () => {
-    expect(ACWithGrants.getRoles()).toEqual(rolesFixture);
+  describe('DENY TESTS', () => {
+    it('permission denied should return false', () => {
```

Comment:
Test cases should read like `it('should return false when permission is denied')`.